src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
While I don't think this explicitly violates the coding standard, I think it would be clearer to use a noun for this variable such as `date`.